packages/flutter/lib/src/widgets/form.dart

```diff
@@ -465,6 +464,20 @@ class FormField<T> extends StatefulWidget {
   /// [FormState.save].
   final FormFieldSetter<T>? onSaved;
 
+  /// An optional property that forces the [FormFieldState] into an error state
+  /// by directly setting the [FormFieldState.errorText] property without
+  /// running the validator function.
+  ///
+  /// When the [forceErrorText] property is provided, the [FormFieldState.errorText]
+  /// will be set to the provided value, causing the form field to be considered
+  /// invalid and display the error message specified.
+  ///
```

Comment:
As mentioned above in another comment, I would mention `validator`. Something like this in a new paragraph maybe:

 > When [validator] is provided, [forceErrorText] will override any error that it returns. [validator] will not be called unless [forceErrorText] is null.